### PR TITLE
[#2470] Hide details of embed project URLs from Butter

### PR DESCRIPTION
**Pull request by mjschranz**

### CODE REVIEW

File: cornfield/app.js [hunk 1]
```diff
@@ -252,16 +232,29 @@ app.post( '/api/publish/:id',
...
              popcornString + data.substring( bodyEndTagIndex );
 
+      // Convert 1234567890 => "kf12oi"
+      var idBase36 = id.toString( 36 ),
+          publishUrl = EMBED_HOSTNAME + '/' + stores.publish.expand( idBase36 ),
+          iframeUrl = EMBED_HOSTNAME + '/' + stores.publish.expand( idBase36 + EMBED_SUFFIX );
+
+      function finished( err ) {
+        if ( err ) {
+          res.json({ error: "internal server error" }, 500);
```
Comment (mjschranz) [inline cornfield/app.js]: Not going to stop reviews based on this but change to single quotes for consistencies sake.

File: src/editor/share-editor.js [hunk 2]
```diff
@@ -173,7 +173,7 @@ define([ "editor/editor", "editor/base-editor", "ui/user-data",
       embedDimensions = embedSize.value.split( "x" );
       embedWidth = embedDimensions[ 0 ];
       embedHeight = embedDimensions[ 1 ];
-      updateEmbed( projectURL.value.replace( "/v/", "/e/" ) );
+      updateEmbed( butter.project.iframeUrl );
```
Comment (ScottDowne) [inline src/editor/share-editor.js:176]: Love this.

Comment (mjschranz) [inline src/editor/share-editor.js:176]: And I love your Gravatar pic.

**Comment by mjschranz** [comment]
close in favor of #1118

**Pull request closed without merging**